proto/decentraland/bff/authentication_service.proto

```diff
@@ -1,39 +0,0 @@
-syntax = "proto3";
```

Comment:
I think we can't delete this files until the migration is over. Do we?

Reply:
Technically, we can, since we only need to merge and release the catalyst, they will get updated in a day or two, and then we can merge unity renderer. That being said.. I guess there is no need to put that pressure on ourselves, so I may restore them in the meantime